index.html

```diff
@@ -177,22 +179,22 @@ <h3 class="connect">LET’S CONNECT</h3>
                 </div>
                 <article class="right-content">
                     <h2 class="lang-info">Languages <span class="more-lang">></span></h2>
-                    <ul class="languages">
-                        <li>
-                            <img src="assets/images/js.png" alt="">
-                            <h4>Javascript</h4>
-                        </li>
-                        <li>
-                            <img src="assets/images/html.png" alt="">
-                            <h4>HTML</h4>
-                        </li>
-                        <li>
-                            <img src="assets/images/css.png" alt="">
-                            <h4>CSS</h4>
-                        </li>
-                    </ul>
 
                     <ul class="tools-container">
+                        <ul class="languages">
+                            <li>
+                                <img src="assets/images/js.png" alt="">
+                                <h4>Javascript</h4>
+                            </li>
+                            <li>
+                                <img src="assets/images/html.png" alt="">
+                                <h4>HTML</h4>
+                            </li>
+                            <li>
+                                <img src="assets/images/css.png" alt="">
+                                <h4>CSS</h4>
+                            </li>
+                        </ul>
```

Comment:
- [OPTIONAL] There is 2 lists here. One that has the Languages/Frameworks/Skills as list items, and other with all the languages inside the Languages list item. It would be better for HTML semantics to have the Languages as the first list item(`li`) of the outer `ul`, and the second `ul` as a sibling of the Languages element. See that the Languages/Skills/Frameworks has the same styling so they would be better to have the same class and same HTML tag.